include/utils.h

```diff
@@ -50,10 +50,19 @@ inline void Log(std::string msg)
   }
 }
 
+namespace smt {
+
 // term helper methods
-void conjunctive_partition(const smt::Term &term, smt::TermVec &out);
+void op_partition(smt::PrimOp o, const smt::Term & term, smt::TermVec & out);
+
+void conjunctive_partition(const smt::Term & term,
+                           smt::TermVec & out,
+                           bool include_bvand = false);
```

Comment:
perhaps, it is good to add comment here about `include_bvand`. Specially, it is important to mention about 1-bit ...

Reply:
Good point!